utils/web_scraper.py

```
@@ -1880,8 +1831,740 @@ def format_window_context_content(content: Dict[str, Any]) -> str:
+    try:
+        twitter_cookies = _get_platform_cookies('twitter')
+        if not twitter_cookies:
+             return {'success': False, 'error': '未配置 config/twitter_cookies.json'}
+             
+        # 提取防伪 CSRF Token。Twitter 必须，否则哪怕有合法 Cookie 也会立刻 401/403
+        ct0 = twitter_cookies.get('ct0') or twitter_cookies.get('CT0', '')
+        if not ct0:
+            logger.warning("Twitter Cookie 中缺少核心字段 ct0，极大可能触发风控拦截")
+        
+        # 官方 Web 客户端通用固化的 Bearer Token
+        bearer_token = 'AAAAAAAAAAAAAAAAAAAAANRILgAAAAAAnNwIyU2%2FGoa3FmBNYDPz%2FzGz%2F2Rnc%2F2bGBDH%2Fc'
```

Ruff (0.15.1)

[error] Possible hardcoded password assigned to: "bearer_token"

(S105)